cmd/crdvalidator/README.md

```diff
@@ -0,0 +1,96 @@
+# crdvalidator
+
+## Summary
+A part of the core value proposition for RukPak is the safe upgrade of a [CustomResourceDefinition](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) (`CRD`). As a result, this repository comes equipped with a "CRD Validator Webhook" (`crdvalidator`), that will validate all `CRD` upgrades created by RukPak. This protects your `BundleInstance` pivots from having potentially dangerous effects such as data loss. 
+
+### How to use
+
+`crdvalidator` will ensure that any `CRD` upgrade is safe as long as it is owned by RukPak. If you would like to disable validation for a `CRD` included in a specific `Bundle`, you will need to explicitly set the `core.rukpak.io/safe-upgrade-validation` annotation to `disabled` within that specific `Bundle`.
```

Comment:
```suggestion
`crdvalidator` will ensure that any `CRD` upgrade is safe as long as the CRD is associated with RukPak. If you would like to disable validation for a `CRD` included in a specific `Bundle`, you will need to explicitly set the `core.rukpak.io/safe-upgrade-validation` annotation to `disabled` within that specific CRD's manifest.
```

I also _really_ think we should include `crd` in the annotation key so that it's more clear what this annotation applies to. Do we feel strongly it should not be included?

Reply:
Sure, that's an easy fix. I'll also add Joe's "true"/"false" suggestion while I'm at it.